oauth2client/service_account.py

```diff
@@ -461,3 +465,94 @@ def create_delegated(self, sub):
         result._private_key_pkcs12 = self._private_key_pkcs12
         result._private_key_password = self._private_key_password
         return result
+
+
+def _datetime_to_secs(utc_time):
+   # time_delta.total_seconds() not supported in Python 2.6
+   epoch = datetime.datetime(1970, 1, 1)
+   time_delta = utc_time - epoch
+   return time_delta.days * 86400 + time_delta.seconds
+
+
+class JWTAccessCredentials(GoogleCredentials):
```

Comment:
@jonparrott: Expound on that? When would gRPC want to do that, and what action would it take based on the outcome of the check?

@kpayson64: Does `create_with_claims(claims)` have to be an instance-scope method of a new public class? Would it be possible to write a function `create_with_claims(google_credentials, claims)` that affords the behavior? (If the respective answers are "yes" and "no" that goes a long way toward justifying the new class' existence.)